guides/common/modules/ref_system-requirements.adoc

```diff
@@ -5,7 +5,7 @@
 The following requirements apply to the networked base operating system:
 
 * x86_64 architecture
-ifeval::["{context}" == "satellite"]
+ifeval::["{Build}" == "satellite"]
```

Comment:
Honestly, "Build" already drives me crazy, I'd find "build" more natural. How about either changing that or introducing an alias: `:build: {Build}` in attributes? :-)

Reply:
This Build is a sneaky thing that just bled in. I'll s/Build/build throughout.